packages/frontend/src/components/SettingsValidator/ValidatorTable/index.tsx

```diff
@@ -119,23 +134,26 @@ const TableValidationItem = forwardRef<
                             </Text>
 
                             {(isChartValidationError(validationError) ||
-                                isDashboardValidationError(
-                                    validationError,
-                                )) && (
-                                <Text fz={11} color="gray.6">
-                                    {getViews(validationError)} view
-                                    {getViews(validationError) === 1 ? '' : 's'}
-                                    {' • '}
-                                    {validationError.lastUpdatedBy ? (
-                                        <>
-                                            Last edited by{' '}
-                                            <Text span fw={500}>
-                                                {validationError.lastUpdatedBy}
-                                            </Text>
-                                        </>
-                                    ) : null}
-                                </Text>
-                            )}
+                                isDashboardValidationError(validationError)) &&
+                                !isDeleted(validationError) && (
```

Comment:
I wonder if here we can just have `!isDeleted(validationError)` 🤔

Reply:
mm,m I don't think so, because you want to know if it is a chart or a dashboard, and not deleted. These views don't work with tables

Reply:
got it, that's what I thought!